src/Components/Notification/index.tsx

```diff
@@ -0,0 +1,192 @@
+import { RightArrow } from '@/Assets/RightArrow';
+import { NOTIFICATIONS } from '@/Constants/queryString';
+import { NotificationResponse } from '@/Hooks/notifications/useNotifications';
+import { useReadNotification } from '@/Hooks/notifications/useReadNotification';
+import { NotificationSSEType } from '@/Types/notifications';
+import { getElapsedTime } from '@/utils/date';
+import { moveToDest } from '@/utils/moveToDest';
+import { useQueryClient } from '@tanstack/react-query';
+import { Link } from 'react-router-dom';
+import { MouseEvent } from 'react';
+import styled from 'styled-components';
+import { media } from '@/Styles/theme';
+import { getNotificationTypeText } from '@/utils/getText';
+
+interface NotificationProps<T extends NotificationSSEType> {
+  /** 알림 타입 */
+  type: T['type'];
+
+  /** 알림 id */
+  notificationId: number;
+
+  /** 알림 Title */
+  title: string;
+
+  /** 알림 상세 내용 */
+  content?: string;
+
+  /** 읽은 기록  */
+  read: boolean;
+
+  /** 페이지 이동에 필요한 params */
+  params: T['params'];
+
+  /** 알림 생성 시간 */
+  createdAt: string;
+}
+
+export const Notification = <T extends NotificationSSEType>(props: NotificationProps<T>) => {
+  const { type, notificationId, title, content, read, params, createdAt } = props;
+
+  const { mutate: readMutate } = useReadNotification(notificationId);
+  const queryClient = useQueryClient();
+
+  const destPagePath = moveToDest(type, params);
+
+  const readNotification = (e: MouseEvent<HTMLAnchorElement>) => {
+    e.stopPropagation();
+
+    /** 이미 읽은 경우, 읽음 처리 X */
+    if (read) return;
+
+    readMutate(notificationId);
+
+    // 얼람 앍움 처리 캐시 반영
+    queryClient.setQueryData(NOTIFICATIONS.NOTIFICATIONS, (prev: { data: { data: NotificationResponse } }) => {
+      const newData = JSON.parse(JSON.stringify(prev));
+
+      const alarmArrIdx = prev?.data?.data?.notification.findIndex(
+        (notification) => notification.notificationId === notificationId,
+      );
+
+      if (alarmArrIdx !== -1) {
+        newData.data.data.notification[alarmArrIdx].read = true;
+      }
+      return newData;
+    });
+  };
```

Comment:
알림페이지에서 사용되는 컴포넌트인데, 이부분도 mutation hook의 onSuccess에서 처리되니까 지워야 할 것 같습니다.
```suggestion
const readNotification = (e: MouseEvent<HTMLAnchorElement>) => {
    e.stopPropagation();
    /** 이미 읽은 경우, 읽음 처리 X */
    if (read) return;
     readMutate(notificationId);
};
```

Reply:
알림 읽음 로직이 루도가 알려요 페이지에서 쓰인다는 걸 놓쳤네요 🙏
import 문도 지워야해서 커밋으로 반영했습니다!
꼼꼼하게 봐주셔서 감사합니다 ☺️